patches/additionalPythonPackages/default.nix

```diff
@@ -460,13 +461,13 @@ in
 
     };
 
-    efel = pythonPackages.buildPythonPackage rec {
+    efel = python27Packages.buildPythonPackage rec {
```

Comment:
What's the reason for this change ?

Reply:
efel is python 2(https://pypi.org/project/efel/) So I want to be explicit about that. Now we have efel python 3 nix package which doesn't work but it is still there.

Reply:
I am not even sure this works. Remove the change on this line and I'll merge

Reply:
done